apps/src/ailab/Ailab.js

```diff
@@ -176,6 +177,18 @@ Ailab.prototype.initMLActivities = function() {
     });
   };
 
+  const logMetric = (eventName, details) => {
+    firehoseClient.putRecord(
+      {
+        study: 'ai-ml',
+        study_group: 'ai-lab',
+        event: eventName,
+        data_json: JSON.stringify({details})
```

Comment:
Do `details` need to be wrapped in braces?  Wouldn't `details` already be an object?